### PR TITLE
using localstorage to store scores and corrected some errors

### PR DESCRIPTION
**Pull request by gnanasekaran-sakthivel**
Fixed few script errors in the console. Then added logic to store the user scores into the local storage.

### CODE REVIEW

File: assets/js/classic-mode.js
Comment (gnanasekaran-sakthivel) [inline assets/js/classic-mode.js:1]: It appears like all the lines are changed. It is because of my auto formatting.

**Review by bhavesh68 — APPROVED** [review]
Thanks for updating Edison! The formatting looks perfect!

**Review by JT-code-dev — APPROVED** [review]
Thank you! This looks great! Great job on the logic for the timing and buttons clicked

**Review by JT-code-dev — COMMENTED** [review]
great job fixing that thank you!

**Pull request merged**